app/views/users/edit.html.erb

```diff
@@ -63,4 +63,22 @@ minlength: User.password_length.min %>
       <% end %>
     </div>
   </div>
+
+  </div>
+ <%= form_for @user, as: :user, url: users_path do |form| %>
+   <br>
+   <h3>Communication Preferences</h3>
```

Comment:
nit: strings should be in en.yml

Reply:
20a081b955c612f77adf10d869faa7704811a0e4